content/docs/getting-started/backends.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Working with Backends
+prev: /docs/getting-started/edit-content
+next: null
+---
+
+You may have noticed that content changes don't persist on refresh. Every time the page loads, the `data` populates with the form's initial values. In order for the CMS to be useful, it needs to track and persist data changes with some sort of backend.
+
+## Loading Content from an external API
+
+For this example, we will use a [fake API](https://jsonplaceholder.typicode.com/) to fetch content and post changes. We will use the `loadInitialValues` function to fetch content.
+
+**src/App.js**
+
+```diff
+//..
+
+function PageContent() {
+  const formConfig = {
+    id: 'tina-tutorial-index',
+    label: 'Edit Page',
+    fields: [
+      //...
+    ],
+-   initialValues: pageData,
++   loadInitialValues() {
++     return fetch(
++       'https://jsonplaceholder.typicode.com/posts/1'
++     ).then((response) => response.json());
++   },
+    onSubmit: async () => {
+      window.alert('Saved!');
+    },
+  }
+
+  //...
+}
+
+//...
+```
+
+Notice how we removed `initialValues` in lieu of `loadInitialValues`, which fetches data asynchronously on form creation.
+
+You can use `initialValues` when your data has already been fetched or defined. For example frameworks like Next.js will load the data for you and pass it to the page component as props.
+
+## Saving Content
+
+Next we'll adjust the `onSubmit` function. When the editor clicks the 'Save' button in the sidebar, `onSubmit` will be called, sending the updated data back to the API:
+
+**src/App.js**
+
+```diff
+function PageContent() {
+  const formConfig = {
+    id: 'tina-tutorial-index',
+    label: 'Edit Page',
+    fields: [
+      //...
+    ],
+    loadInitialValues() {
+      return fetch(
+        'https://jsonplaceholder.typicode.com/posts/1'
+      ).then(response => response.json())
+    },
+-   onSubmit: async () => {
+-     window.alert('Saved!');
+-   },
++   onSubmit(formData) {
++    fetch('https://jsonplaceholder.typicode.com/posts/1', {
```

Comment:
```suggestion
+    return fetch('https://jsonplaceholder.typicode.com/posts/1', {
```

if we return a promise here the loading state on the save button will work properly (hard to notice since the API responds so fast)